nf_core/modules/lint/main_nf.py

```diff
@@ -236,25 +236,30 @@ def check_process_section(self, lines, fix_version, progress_bar):
 
     # Check that process labels are correct
     correct_process_labels = ["process_single", "process_low", "process_medium", "process_high", "process_long"]
-    process_label = [l for l in lines if l.lstrip().startswith("label")]
-    if len(process_label) > 0:
-        try:
-            process_label = re.search("process_[A-Za-z]+", process_label[0]).group(0)
-        except AttributeError:
-            process_label = re.search("'([A-Za-z_-]+)'", process_label[0]).group(0)
-        finally:
-            if not process_label in correct_process_labels:
+    all_labels = [l for l in lines if l.lstrip().startswith("label ")]
+    bad_labels = []
+    good_labels = []
+    if len(all_labels) > 0:
+        for label in all_labels:
+            label = re.match("^label\s+([a-zA-Z0-9_-]+)", label).group(1)
+            if label not in correct_process_labels:
+                bad_labels.append(label)
+            else:
+                good_labels.append(label)
```

Comment:
```suggestion
            if label in correct_process_labels:
                good_labels.append(label)
```
`bad_labels` is not used in the rest of the function, I'm guessing it can be removed?

Reply:
There's more coming. I think it's worth adding more warnings/checks for e.g. duplicate labels

Reply:
Use of `bad_labels` in latest commit